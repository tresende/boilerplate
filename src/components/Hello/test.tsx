import { render } from 'utils/test-utils'
import Component from '.'

describe('<Component />', () => {
  it('should render component correctly', () => {
    const { container } = render(<Component />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
