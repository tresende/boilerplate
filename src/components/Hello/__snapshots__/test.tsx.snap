// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Component /> should render component correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-right: 2.4rem;
}

<div
  class="c0"
/>
`;
